static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss

```diff
@@ -0,0 +1,148 @@
+@mixin immersiveCommon() {
+    .fc-item__kicker {
+        display: block;
+    }
+
+    .fc-item__container {
+        overflow: hidden;
+    }
+
+    .fc-item__content {
+        background-color: $immersive-garnett-main-1;
+        z-index: 1;
```

Comment:
Are we able to remove this `z-index`? Would be great if we didn't need to rely on it!